daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -378,6 +378,9 @@ object SExpr {
     def execute(machine: Machine): Unit = {
       machine.pushKont(KTryCatchHandler(machine, handler))
       machine.ctrl = body
+      machine.withOnLedger("SETryCatch") { onLedger =>
```

Comment:
same here

Reply:
But not here (without some reorganisation) because this is an expression, not a builtin.